src/status_im/ui/screens/network_settings/events.cljs

```diff
@@ -0,0 +1,42 @@
+(ns status-im.ui.screens.network-settings.events
+  (:require [re-frame.core :as re-frame]
+            [taoensso.timbre :as log]
+            [status-im.ui.screens.navigation :as navigation]
+            [status-im.fleet.core :as fleet-core]
+            [status-im.utils.handlers :as handlers]
+            [status-im.utils.fx :as fx]))
+
+(defn- show-popup [title msg]
+  {:utils/show-popup [title msg nil]})
+
+(defn les-not-supported-warning [fleet]
+  (let [message
+        (str (name fleet)
+             " does not support LES!\n"
+             "Please, select one of the supported fleets:"
+             (map name fleet-core/fleets-with-les))]
+    (show-popup "LES not supported" message)))
+
+(defn experimental-warning []
+  (let [message "Use at your own risk!"]
+    (show-popup "LES support is experimental!" message)))
+
+(fx/defn open-network-details
+  [cofx network show-warning?]
+  (fx/merge cofx
+            (when show-warning?
+              (experimental-warning))
```

Comment:
use the show-popup map directly
especially here since the second arg of fx/merge is a special case where fx/merge checks if it is a map and uses it as an initial fx.
here the function is returning a map before fx/merge is called, but that might not be obvious for everybody since fx/defn functions return a function when they are called without a cofx argument.
using the map with show-popup effect directly is more explicit